sixtools/tests/test_preliminary_processing.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import numpy as np
+from sixtools import preliminary_processing as pre_proc
+
+
+@pytest.fixture
+def data():
+    # generate a list of 2D numpy arrays.
+    out_list = []
+    for i in range(0, 3):
+        out_list.append(np.arange(1, 26).reshape(5, 5))
+
+    yield out_list
```

Comment:
This construction is odd. I think this should either be:

```py
    for i in range(0, 3):
        yield np.arange(1, 26).reshape(5, 5)
```

or

```py
    out_list = []
    for i in range(0, 3):
        out_list.append(np.arange(1, 26).reshape(5, 5))

    return out_list
```

Reply:
it should yield a list of 3 identical arrays at each point. not a single array (as in your first 
ie.
`next (instance)`
`[array1, array2, array3]`
suggestion). It needs to be a generator, so your secodn sugestion is also not what is needed

Reply:
I should add for clarification, this imitates the output structure for db[xxxx].data(....)